CONTRIBUTING.md

```diff
@@ -1,7 +1,104 @@
+and then it will enter a Final Comment Period. If the pull request passes the final comment period without opposition,
+the PR will be merged. Otherwise, the PR will return to being in review.
+
+The exact number of reviews needed, and the length of the Final Comment Period, varies depending on the scope and
+complexity of the pull request. The numbers for each category are listed below.
+
+> **Minor changes such as typo fixes may bypass the process.**
+
+### Classifications
+PRs can be classified by the type of the changes (e.g. new mappings, toolchain updates, mappings refactor, documentation),
+by the type of Minecraft version they target (`release` or `snapshot`), and by their size. This classification is
+done by the Mappings Team and Mappings Triage, and is used to determine the number of reviews needed and the length of
+the FCP. GitHub PR labels should be used to indicate which category a PR belongs to.
+
+#### Change type
+Note that for the required reviews, approvals from members of the Mappings Team may be considered as Triage reviews.
+
+##### New mappings
+- Small PR
+  - **Required Approvals: 2**, at least 1 from the Mappings Team and 1 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 1 day
+    - Releases: 3 days
+- Large PR
+  - **Required Approvals: 3**, at least 1 from the Mappings Team and 2 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 3 days
+    - Releases: 7 days
+
+##### Mapping refactors
+Typo fixes are *not* considered refactor changes.
+
+- Small PR
+  - **Required Approvals: 3**, at least 2 from the Mappings Team and 1 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 5 days
+    - Releases: 7 days
+- Large PR
+  - **Required Approvals: 4**, at least 2 from the Mappings Team and 2 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 7 days
+    - Releases: 10 days
+
+##### Toolchain changes
+Changes to the buildscript, build system, or other tools used in QM are considered toolchain changes.
+
+- Small PR
+  - **Required Approvals: 1**, from the Mappings Team
+  - **Final Comment Period:**
+    - Snapshots: 2 days
+    - Releases: 4 days
+- Large PR
+  - **Required Approvals: 3**, from the Mappings Team
+  - **Final Comment Period:**
+    - Snapshots: 7 days
+    - Releases: 10 days
+
+##### Documentation
+- Small PR
+  - **Required Approvals: 2**, at least 1 from the Mappings Team and 1 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 3 days
+    - Releases: 7 days
+- Large PR
+  - **Required Approvals: 3**, at least 1 from the Mappings Team and 2 from Mappings Triage
+  - **Final Comment Period:**
+    - Snapshots: 6 days
+    - Releases: 10 days
+
+#### Target version
+Affects only the length of the FCP.
+- Release PRs are those that target a release of Minecraft, such as `1.17`, `1.17.1` or `1.18`
+- Snapshot PRs are those that target a snapshot of Minecraft, such as `21w44a`, `1.17.1-pre1` or `1.18-rc4`
```

Comment:
I personally think Release Candidate (such as `1.18.1-rc2`)versions should be classed as release versions, due to the lack of difference between each of the Release Candidates pushed by Mojang. Though that is just my opinion and this review can be disregarded if need be. Other than that all good.